packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -557,6 +563,9 @@ export class ColorPicker
       context: this.hueSliderRenderingContext,
       bounds: this.hueSliderRenderingContext.canvas.getBoundingClientRect(),
     };
+    if (event.target === this.hueScopeNode) {
+      return;
+    }
     this.captureHueSliderColor(offsetX);
```

Comment:
Color should still be captured when the hue scope is the event target. Each pixel along the hue slider can update the underlying color.

This also applies to the color field scope.

On a related note, since the scopes are mostly for keyboard interaction, should we disable pointer events on them to simplify the code a bit? Pointer interactions are already handled and we didn't have to worry about pointer events on the scopes since they were 0x0.